flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/DefaultScheduler.java

```diff
@@ -669,6 +704,11 @@ public SchedulingTopology getSchedulingTopology() {
         public Optional<TaskManagerLocation> getStateLocation(ExecutionVertexID executionVertexId) {
             return stateLocationRetriever.getStateLocation(executionVertexId);
         }
+
+        @Override
+        public Set<AllocationID> getAllocationsToReserve() {
```

Comment:
Maybe `getReservedAllocations`?

Reply:
Agreed. `reservedAllocations` is more accurate.

Reply:
done.